src/helm/benchmark/scenarios/scenario.py

```diff
@@ -55,7 +56,11 @@ class Input:
     The input of an `Instance`.
     """
 
-    text: str
+    text: str = ""
```

Comment:
Semantically, it would be more accurate if `text` were `Optional[str] = None`

Reply:
I tried to update it to `Optional[str]`, but type checking breaks with `40 errors in 20 files`.